functions/nvm.fish

```diff
@@ -1,11 +1,17 @@
 function nvm -d "Node version manager"
-  if test -e $NVM_DIR/nvm.sh
+  if available brew; and brew --prefix nvm > /dev/null
+    set nvm_runnable (brew --prefix nvm)
```

Comment:
See the note above about using global variables, would save us some seconds in avoiding running this check every time the command `nvm` is issued.